graph/resolver/user.resolvers.go

```diff
@@ -93,6 +94,63 @@ func (r *userResolver) Devices(ctx context.Context, obj *model.User) ([]*model.U
 	return resp, nil
 }
 
+func (r *userResolver) Discussions(ctx context.Context, obj *model.User) ([]*model.Discussion, error) {
+	authedUser := auth.GetAuthedUser(ctx)
+	if authedUser == nil {
+		return nil, fmt.Errorf("Need auth")
+	}
+
+	// Do we want a mod override?
+	if authedUser.UserID != obj.ID {
+		return nil, fmt.Errorf("unauthorized")
+	}
+
+	return r.DAOManager.GetDiscussionAccessByUserID(ctx, authedUser.UserID)
```

Comment:
We should consider sorting these as well... Perhaps we even want to return a connection with discussions instead of a list? Let's add this as an additional step for the future though...

Reply:
https://github.com/nedrocks/delphisbe/issues/33